sources/packages/api/src/esdc-integration/msfaa-integration/msfaa-response.service.ts

```diff
@@ -105,6 +118,30 @@ export class MSFAAResponseService {
     }
   }
 
+  /**
+   * Process the cancelled record of the MSFAA response file
+   * by updating the columns of MSFAA Numbers table
+   * @param cancelledRecord MSFAA received record with
+   * cancelledDate and newIssusingProvince.
+   */
+  private async processCancelledRecords(
```

Comment:
the method can have a singular name, when I initially read the name, it looked it it process multiple records(cancellation records).